internal/manifest/manifest.go

```diff
@@ -126,21 +126,61 @@ func (p Policy) Hash() HexString {
 	return NewHexString(hashBytes[:])
 }
 
+// Validate checks the validity of all fields in the reference values.
+func (r ReferenceValues) Validate() error {
+	if r.SNP.MinimumTCB.BootloaderVersion == nil {
+		return fmt.Errorf("field BootloaderVersion in manifest cannot be empty")
+	} else if r.SNP.MinimumTCB.TEEVersion == nil {
+		return fmt.Errorf("field TEEVersion in manifest cannot be empty")
+	} else if r.SNP.MinimumTCB.SNPVersion == nil {
+		return fmt.Errorf("field SNPVersion in manifest cannot be empty")
+	} else if r.SNP.MinimumTCB.MicrocodeVersion == nil {
+		return fmt.Errorf("field MicrocodeVersion in manifest cannot be empty")
+	}
+
+	if len(r.TrustedMeasurement) != 96 {
```

Comment:
Nit: we should replace all raw 48s with [`abi.MeasurementSize`](https://pkg.go.dev/github.com/google/go-sev-guest/abi#pkg-constants).